packages/acceptance-tests/pageObjects/Page.ts

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+
+import { Page as PWPage } from '@playwright/test'
+import config from 'config'
+
+const selectors: Record<string, string> = {
+	spinners: '.waitSpinner',
+	body: 'body',
+}
+
+export class Page {
+	public constructor(private _page: PWPage) {}
+
+	protected get page() {
+		return this._page
+	}
+
+	protected get rootUrl() {
+		return config.get('url')
+	}
+
+	protected async waitForLoad() {
+		await this.page.waitForSelector(selectors.body)
+		await this.page.waitForSelector(selectors.spinners, { state: 'detached' })
+	}
+
+	protected async open(path: string): Promise<void> {
+		this.page.goto(`${this.rootUrl}/${path}`)
```

Comment:
We could avoid most of the overrides in classes that extend Page by adding a path property to subclasses and then   using it automatically if it is not passed to this method (i.e., `const p = this.path || path`).

Reply:
Some of these are parameterized, which precludes this, but I added the ability to define a default pagePath for static paths.